www/delivery_dev/ck.php

```diff
@@ -76,11 +79,37 @@
         $creativeId[$i] = 0;
     }
     if (($adId[$i] > 0 || $adId[$i] == -1) && ($conf['logging']['adClicks']) && !(isset($_GET['log']) && ($_GET['log'] == 'no'))) {
+        // If no-click/redirect on inactive banners is enabled, check to see
+        // if the banner is active. If not, exit click processing at this point,
+        // without recording the click or performing redirection
+        if (isset($GLOBALS['conf']['logging']['blockInactiveBannerClicks'])) {
+            $exitProcessing = true;
+            $aZoneLinkedAds = MAX_cacheGetZoneLinkedAds($zoneId);
+            $aAds = array_merge($aZoneLinkedAds['xAds'], $aZoneLinkedAds['ads'], $aZoneLinkedAds['lAds']);
```

Comment:
This part would be replaced with a simpler status check, but, for the record, $aZoneLinkedAds['ads'] contains ads indexed by priority level, so array_merge isn't producing the expected result.